src/defaults.mligo

```diff
@@ -3,6 +3,8 @@
 
 #include "types.mligo"
 
+let frozen_token_id: nat = 0n
```

Comment:
This isn't really selectable tho (even tho it is in storage), well it can't be in all cases anyway.

As per FA2 we need to know that when there is only one `token_id` in the contract this has to be `0`.

We could either check that this is the case in `default_storage` (but than it's hard to say what to do when it is not the case) or not allow the decision at all, which is pretty fair.

IMO we can remove this and in `default_storage`  just set `frozen_token_id = 0n` instead.

(well done in removing this from the `types` module tho)

Reply:
> IMO we can remove this and in `default_storage` just set `frozen_token_id = 0n` instead

I left this as a separate binding because it's used in `total_supply = Map.literal [ (frozen_token_id, 0n) ];`. Although it isn't configurable, having `(0n, 0n)` in total supply harms readability IMO.

Reply:
Oh I see, makes sense to me too, no problem then :ok_hand: